packages/nimble-components/src/text-area/index.ts

```diff
@@ -26,6 +27,99 @@ export class TextArea extends FoundationTextArea {
      */
     @attr
     public appearance: TextAreaAppearance = TextAreaAppearance.outline;
+
+    /**
+     * A message explaining why the value is invalid.
+     *
+     * @public
+     * @remarks
+     * HTML Attribute: error-text
+     */
+    @attr({ attribute: 'error-text' })
+    public errorText?: string;
+
+    /**
+     * Whether to display the error state.
+     *
+     * @public
+     * @remarks
+     * HTML Attribute: error-visible
+     */
+    @attr({ attribute: 'error-visible', mode: 'boolean' })
+    public errorVisible = false;
+
+    /**
+     * The width of the vertical scrollbar, if displayed.
+     * @internal
+     */
+    @observable
+    public scrollbarWidth = 0;
+
+    private resizeObserver?: ResizeObserver;
+    private updateScrollbarWidthQueued = false;
+
+    /**
+     * @internal
+     */
+    public override connectedCallback(): void {
+        super.connectedCallback();
+        this.resizeObserver = new ResizeObserver(this.onResize);
+        this.resizeObserver.observe(this);
+    }
+
+    /**
+     * @internal
+     */
+    public override disconnectedCallback(): void {
+        this.resizeObserver?.disconnect();
+    }
+
+    /**
+     * @internal
+     */
+    public onTextAreaInput(): void {
+        this.handleTextInput();
+        this.queueUpdateScrollbarWidth();
+    }
+
+    // If a property can affect whether a scrollbar is visible, we need to
+    // call queueUpdateScrollbarWidth() when it changes. The exceptions are
+    // properties that affect size (e.g. height, width, cols, rows), because
+    // we already have a ResizeObserver handling those changes.
+
+    /**
+     * @internal
+     */
+    public placeholderChanged(): void {
+        this.queueUpdateScrollbarWidth();
+    }
+
+    /**
+     * @internal
+     */
+    public override valueChanged(previous: string, next: string): void {
+        super.valueChanged(previous, next);
+        this.queueUpdateScrollbarWidth();
+    }
+
+    private readonly onResize = (): void => {
+        this.updateScrollbarWidth();
+    };
```

Comment:
just make this a method

Reply:
I was trying to avoid an "unbound-method" lint error, but I can also just bind the method where I pass it to the ResizeObserver.

Reply:
This is a nit, but please prefer the pattern mentioned above / in the styleguide

Reply:
No problem. Fixed.